github/misc.go

```diff
@@ -129,13 +129,13 @@ func (c *Client) ListCodeOfConductByKey(ctx context.Context, key string) (*Condu
 	// TODO: remove custom Accept header when this API fully launches.
 	req.Header.Set("Accept", mediaTypeCodesOfConductPreview)
 
-	conduct := new(Conduct)
-	resp, err := c.Do(ctx, req, &conduct)
+	codeOfConduct := new(CodeOfConduct)
```

Comment:
I'd suggest using a shorter variable name here too. `coc := new(CodeOfConduct)` or `code := new(CodeOfConduct)` would be completely fine, and likely more readable.